.github/workflows/ci.yaml

```diff
@@ -193,6 +193,12 @@ jobs:
       - run: shfmt -i 2 -w $(shfmt -f .)
       - run: git diff --exit-code
 
+  link-liveness:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - run: grep -R -h -o --exclude-dir=bazel-* --exclude=*test.cpp --exclude=ci.yaml 'https://[^)(}{", ]*' | grep -v '^https://$' | xargs wget --spider
```

Comment:
Maybe `--recursive --no-filename --only-matching` instead of the shorthand? Makes looking at this and understanding it without docs easier.

You can probably also skip excluding `bazel-*` since those dirs are only created after the first bazel invocation, and maybe add `--exclude=WORKSPACE` since all of those links are checked when we build anyway.

Reply:
ah yeah, that's a good point